packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -1443,6 +1437,29 @@ describe("IModelTransformerHub", () => {
     expect(count(sourceDb, PhysicalModel.classFullName)).to.equal(1);
     expect(count(targetDb, PhysicalModel.classFullName)).to.equal(1);
 
+    sourceDb.elements.deleteElement(secondCopyOfSubjectId);
+    sourceDb.saveChanges();
+    await sourceDb.pushChanges({ description: "deleted the second copy of the subject"});
+    const startChangeset = sourceDb.changeset;
```

Comment:
add a comment like `// readd the subject in a separate changeset`

Reply:
done